tests/features/piwik.feature

```diff
@@ -1,20 +1,89 @@
-  When I run drush "pm-enable bootstrap -y"
-  And I run drush "vset theme_default bootstrap"
-  Then I am on the homepage
-  And the response should contain "{\"utility\":\"piwik\",\"siteID\":\"\",\"sitePath\":[\"\"],\"is404\":false,\"instance\":\"\"}"
-  When I run drush "vset theme_default ec_resp"
-  Then I am on the homepage
-  And the response should contain "{\"utility\":\"piwik\",\"siteID\":\"\",\"sitePath\":[\"\"],\"is404\":false,\"instance\":\"\"}"
+  Background:
+    Given these modules are enabled
+      | modules            |
+      | nexteuropa_piwik   |
+    And the nexteuropa_piwik module is configured to use advanced PIWIK rules
+    And I am logged in as a user with the "administrator" role
 
+  @wip
+  Scenario: Administrator user can check Piwik Script with the theme Bootstrap
+    Given the module is enabled
+      | modules            |
+      | nexteuropa_piwik   |
+    Given I am logged in as a user with the 'administrator' role
+    When I run drush "pm-enable bootstrap -y"
+    And I run drush "vset theme_default bootstrap"
+    And the cache has been cleared
+    Then I am on the homepage
+    And the response should contain "{\"utility\":\"piwik\",\"siteID\":\"\",\"sitePath\":[\"\"],\"is404\":false,\"instance\":\"\"}"
+    When I run drush "vset theme_default ec_resp"
+    And the cache has been cleared
+    Then I am on the homepage
+    And the response should contain "{\"utility\":\"piwik\",\"siteID\":\"\",\"sitePath\":[\"\"],\"is404\":false,\"instance\":\"\"}"
+
+  Scenario: View advanced PIWIK rules.
+    Given the following PIWIK rules:
+      | Rule language | Rule path       | Rule path type | Rule section         |
+      | all           | ^admin/*        | regexp         | Regexp based section |
+      | en            | content/test    | direct         | Direct path section  |
+    When I go to "/admin/config/system/webtools/piwik/advanced_rules"
+    Then I see an overview with the following PIWIK rules:
+      | Rule language | Rule path       | Rule path type | Rule section         |
+      | all           | ^admin/*        | regexp         | Regexp based section |
+      | en            | content/test    | direct         | Direct path section  |
+
+  @javascript
+  Scenario: Add a PIWIK rule.
+    When I go to "/admin/config/system/webtools/piwik/advanced_rules"
+    And I click "Add piwik rule"
+    And I fill in "Site section" with "Regexp based section"
+    And I select "English" from "Language"
+    And I select the radio button "Path based on regular expression" with the id "edit-rule-type-regexp"
+    And I wait for AJAX to finish
+    And I fill in "Site path" with "^admin/*"
+    And I press the "Save" button
+    Then I see an overview with the following PIWIK rules:
+      | Rule language | Rule path       | Rule path type | Rule section         |
+      | en            | ^admin/*        | regexp         | Regexp based section |
+
+  Scenario: Remove a PIWIK rule.
```

Comment:
In this scenario, we need to check that the deleted rule does not appear in the page.